spec/builders/unpublish_email_builder_spec.rb

```diff
@@ -0,0 +1,65 @@
+# rubocop:disable Layout/EmptyLinesAroundArguments
```

Comment:
Same thing for this https://github.com/alphagov/email-alert-api/pull/655/review_comment/create#r209238087

Reply:
👍